### ISSUE TITLE
[11.0] Wrong Calculation of Quotation/Opportunities in Sales Channels Reporting 

### ISSUE THREAD

**Issue by sswapnesh** [odoo/odoo#24050]
Impacted versions: 11.0

Steps to reproduce: CRM --> Reporting  --> Sales Channels --> Go to Quotation for Any Sales Team

Current behavior: Different total on Dashboard and List view
For America Channel, it is  315.00.

![](https://image.prntscr.com/image/2rLpH6NaQjqcsFp7EjN-1Q.png)
In List view, Total is 405.00

![](https://image.prntscr.com/image/9k3DSW4fQ6yFy1K8nS2DNQ.png)

Expected behavior: Amount should be Same.

Video/Screenshot link (optional):

Note: Multi-currency is not enabled.

**Comment by sswapnesh** [comment]
Video: https://youtu.be/AXT_tNFEauI
Opw 2009808 (For v12)

**Comment by MohsenWaleed** [comment]
@sswapnesh Fix #34391 not solve the issue of wrong calculation and the calculation still showing wrong values.

**Comment by sswapnesh** [comment]
You are right, and I have already mentioned this on the Related PR (With all details and screenshot if that Build) and Ticket too but it seems they just want to close the Tickets w/o proper Fix.

**Comment by MohsenWaleed** [comment]
I got the cause of thee wrong calculation, the company currency is USD so the wrong calculation happen because of the wrong rate for USD currency as you can see in the below screen shot 

![image](https://user-images.githubusercontent.com/24973578/60354187-5a620300-99d4-11e9-95b5-437c59534f62.png)

Just change the rate to 1 for all rates and go to sales report and you will see a correct calculation. 

check it in the below runbot after fixing the issue:
http://549387-11-0-opw-2009808-refix-sig-ba795e.runbot13.odoo.com/web?debug#view_type=kanban&model=crm.team&action=288

**Comment by sswapnesh** [comment]
See https://github.com/odoo/odoo/pull/34391#issuecomment-505954552

**Comment by MohsenWaleed** [comment]
@sswapnesh I see it but there is no issue in the code. as I mentioned above the issue in the USD currency rate.

please check it now:

![image](https://user-images.githubusercontent.com/24973578/60354718-86ca4f00-99d5-11e9-9a7d-1ad46d7271af.png)


![image](https://user-images.githubusercontent.com/24973578/60354674-71552500-99d5-11e9-9729-94dee06459a3.png)

**Comment by MohsenWaleed** [comment]
There is only an issue in the decimals value which need to be fixed

https://user-images.githubusercontent.com/16624719/60198028-4b594480-985e-11e9-9278-67b2a763b3f7.jpg


https://user-images.githubusercontent.com/16624719/60198067-662bb900-985e-11e9-9213-e745920bae4f.jpg

**Comment by sswapnesh** [comment]
And What about other two points?

**Comment by MohsenWaleed** [comment]
This is an issue need to be fixed from the query in sale_report.py file

**Comment by sswapnesh** [comment]
> @sswapnesh I see it but there is no issue in the code. as I mentioned above the issue in the USD currency rate.
> 
> please check it now:
> 
> ![image](https://user-images.githubusercontent.com/24973578/60354718-86ca4f00-99d5-11e9-9a7d-1ad46d7271af.png)
> 
> ![image](https://user-images.githubusercontent.com/24973578/60354674-71552500-99d5-11e9-9729-94dee06459a3.png)
> 
> Please check it in the same runbot which you worked on
> 
> http://549387-11-0-opw-2009808-refix-sig-ba795e.runbot13.odoo.com/web?debug#view_type=kanban&model=crm.team&action=288

See the Inconsistency in the Filter (Check this where Atleast one Quotation is there in "Quotation Sent" state.

**Comment by MohsenWaleed** [comment]
The calculation depend on draft or sent Quotations so the calculation is correct. the sent quotation still a Quotation even if it's sent.

**Comment by MohsenWaleed** [comment]
> And What about other two points?

The two points issue in website figure cause because of the field is Integer 

![image](https://user-images.githubusercontent.com/24973578/60357015-2b02c480-99db-11e9-98d9-ee584bba695c.png)

**Comment by C3POdoo** [comment]
Dear @sswapnesh,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
